worlds/wargroove2/Client.py

```diff
@@ -0,0 +1,858 @@
+                                  f"Looked in \"{appdata_wargroove}\".\n"
+                                  f"If you haven't yet booted the game at least once, boot Wargroove 2 "
+                                  f"and then close it to attempt to fix this error.\n"
+                                  f"If the AppData folder above seems wrong, try setting the "
+                                  f"\"save_directory\" value in your local options file "
+                                  f"to the AppData folder containing your Wargroove 2 saves.")
+
+        root_directory = game_options["root_directory"]
+        if not os.path.isfile(os.path.join(root_directory, "win64_bin", "wargroove64.exe")):
+            print_error_and_close(f"WargrooveClient couldn't find wargroove64.exe in "
+                                  f"\"{root_directory}/win64_bin/\".\n"
+                                  f"Unable to infer required game_communication_path.\n"
+                                  f"Please verify the \"root_directory\" value in your local "
+                                  f"options file is set correctly.")
+        self.game_communication_path = os.path.join(root_directory, "AP")
+        if not os.path.exists(self.game_communication_path):
+            os.makedirs(self.game_communication_path)
+        self.remove_communication_files()
+        atexit.register(self.remove_communication_files)
+
+        mods_directory = os.path.join(appdata_wargroove, "mods", "ArchipelagoMod")
+        save_directory = os.path.join(appdata_wargroove, "save")
+
+        # Wargroove 2 doesn't always create the mods directory, so we have to do it
+        if not os.path.isdir(mods_directory):
+            os.makedirs(mods_directory)
+        resources = ["data/mods/ArchipelagoMod/maps.dat",
+                     "data/mods/ArchipelagoMod/mod.dat",
+                     "data/mods/ArchipelagoMod/modAssets.dat",
+                     "data/save/campaign-45747c660b6a2f09601327a18d662a7d.cmp",
+                     "data/save/campaign-45747c660b6a2f09601327a18d662a7d.cmp.bak"]
+        file_paths = [os.path.join(mods_directory, "maps.dat"),
+                      os.path.join(mods_directory, "mod.dat"),
+                      os.path.join(mods_directory, "modAssets.dat"),
+                      os.path.join(save_directory, "campaign-45747c660b6a2f09601327a18d662a7d.cmp"),
+                      os.path.join(save_directory, "campaign-45747c660b6a2f09601327a18d662a7d.cmp.bak")]
+        for resource, destination in zip(resources, file_paths):
+            file_data = pkgutil.get_data("worlds.wargroove2", resource)
+            if file_data is None:
+                print_error_and_close("Wargroove2Client couldn't find Wargoove 2 mod and save files in install!")
+            with open(destination, 'wb') as f:
+                f.write(file_data)
+
+    def on_deathlink(self, data: typing.Dict[str, typing.Any]) -> None:
```

Comment:
```suggestion
    def on_deathlink(self, data: Dict[str, Any]) -> None:
```
These are already imported and used elsewhere. Also the import of the `typing` module is only used here and could be removed.